.github/workflows/test.yml

```diff
@@ -0,0 +1,22 @@
+name: TDR Run API tests
+on:
+  pull_request_target:
+  push:
+    branches-ignore:
+      - master
+      - release-*
+permissions:
+  id-token: write
+  contents: read
+jobs:
+  test:
+    uses: nationalarchives/tdr-github-actions/.github/workflows/tdr_test.yml@main
+    with:
+      repo-name: tdr-transfer-frontend
+      test-command: |
+        npm --prefix npm ci
```

Comment:
Do we need the npm linting commands in here as well, or is that covered by the `npm run checks` command?

Reply:
It's covered in the `run checks` I've updated the package json to add the checks command which runs the linting and the tests.